minikube/worker-deployment.yaml

```diff
@@ -18,14 +18,35 @@ spec:
     spec:
       containers:
         - env:
+            - name: BROKER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: broker-secret
+                  key: password
+            - name: BROKER_HOST
+              value: broker
+            - name: BROKER_PORT
+              value: "6379"
             - name: BROKER_URL
-              value: redis://broker:6379
+              value: "redis://:$(BROKER_PASSWORD)@$(BROKER_HOST):$(BROKER_PORT)"
             - name: CONTAINER_MODE
               value: worker
             - name: DB_URL
               value: postgresql+psycopg://postgres:passwd@postgres/qunicorn
             - name: NUMBA_CACHE_DIR
               value: /app/cache
+            - name: QMWARE_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: qmware-secret
+                  key: QMWARE_API_KEY
+            - name: QMWARE_API_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: qmware-secret
+                  key: QMWARE_API_KEY_ID
+            - name: QMWARE_URL
+              value: http://10.186.9.12:9003/
```

Comment:
We should remove the IP in the public Repository and make a comment to fill in before usage

Reply:
pardon me, it was indeed an unintentional mistake